test/docs.js

```diff
@@ -111,10 +111,17 @@ describe('documentation', function() {
       global: global
     };
 
+    function FakeExpress() {
+      return {
+        get: function() {}
```

Comment:
Ah, I never realized that before, that kinda stinks, but I suppose it can't be helped.